### PR TITLE
Add direct access to editing objects

### PR DESCRIPTION
**Pull request by fredj**
fixes https://github.com/camptocamp/c2cgeoportal/issues/2090

### CODE REVIEW

File: core/src/script/CGXP/plugins/Editing.js [hunk 1]
```diff
@@ -485,6 +492,57 @@ cgxp.plugins.Editing = Ext.extend(gxp.plugins.Tool, {
         this.initKeyEvent();
     },
 
+    /** private: method[applyState]
+     */
+    applyState: function(state) {
+        if (state.layer !== undefined) {
+            if (state.id !== undefined) {
+                var protocol = new OpenLayers.Protocol.HTTP({
+                    url: this.layersURL + state.layer,
```
Comment (sbrunner) [inline core/src/script/CGXP/plugins/Editing.js]: We should read only one feature: http://trac.mapfish.org/trac/mapfish/wiki/MapFishProtocol#Readonefeature

Comment (fredj) [inline core/src/script/CGXP/plugins/Editing.js]: yes, good remark. thanks

**Comment by fredj** [comment]
ready for review, thanks

**Comment by sbrunner** [comment]
Looks good, merge :-)

**Comment by fredj** [comment]
Thanks for the review !

**Pull request merged**